README.md

```diff
@@ -415,6 +417,36 @@ onfidoOut.tearDown()
   }
   ```
 
+- **`crossDeviceClientIntroProductName {String} optional`**
```

Comment:
Good suggestion. I didn't put it there as the documentation in "Flow customisation" section seemed to be all specific to SDK steps configurations and wanted to avoid any confusion about where to use it in the SDK configuration, but this could work.

Reply:
Looks good! Thanks